include/arch/arm/arch.h

```diff
@@ -243,6 +249,11 @@ extern "C" {
 					 MPU_REGION_SU_RWX)
 #define K_MEM_PARTITION_P_RX_U_RX	(MPU_REGION_READ | MPU_REGION_EXEC | \
 					 MPU_REGION_SU)
+
+#define K_MEM_PARTITION_IS_WRITABLE(attr) \
+	((attr) & (MPU_REGION_WRITE | MPU_REGION_SU_RW))
```

Comment:
Can't simply use OR operation to check here, since the P_RO_U_RO attribute will also make this statement
to be Ture.

Reply:
Done.